### PR TITLE
Desktop portfolio

### PR DESCRIPTION
**Pull request by dazehinn**
Added styles for the desktop version of the portfolio. Index.html had to be modified a little to work with the stylesheet.

### CODE REVIEW

File: responsive-style.css [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    width: 544px;
+    height: fit-content;
+    align-items: flex-start;
+  }
+
+  .snapshot {
+    width: 554px;
+    border-radius: 1px;
+  }
+
```
Comment (Abenezer-Tilahun) [inline responsive-style.css]: - [ ] As stated the project requirement You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly make sure that work section is responsive for  desktop screen size(between 768 and 1440), Please adjust the width.

Your Project Work section Look likes this : 

Capture

The Figma Design Work section Look likes this : 

Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @dazehinn  , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linters error
✔️ You used correct GitHub Flow.
✔️ Professional README
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ] Good job for documenting your README File Professional, But I recommend to include the the "Getting Started " section and  "Future Features" section, including 1-3 potential features to be added.and All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in a your's repo.

_Check the comments below the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: responsive-style.css (hunk 1 above)
Comment (zieeco) [reply responsive-style.css]: - Hey, great job so far and I'm approving your project based on the changes required by the previous reviewer.

Please, kindly note the following:

**[OPTIONAL]**

- [ ] In the desktop version, the second and fourth cards are reversed. You can check the Figma design to see this as though the previous reviewer missed this.
- [ ] Your card is not having a rounded border as it is sharp. Please, try to fix all these issues to improve your design as much as you can.
- [ ] You've not used the correct background image for the contact form on the desktop version. Your image is still that of the mobile version.
- [ ] Your language images are too big in width compared to the Figma design. Let's try and improve on this.f
- [ ] The social media icon in your about section are not having enough space in between them compared to the Figma design. Please, do fix this issue when you got time to improve your design.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @dazehinn, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
_
_
_Check the comments under the review_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**